### DISCUSSION TITLE
Bug?

### DISCUSSION THREAD

**Discussion by LordSith77** [advplyr/audiobookshelf#504] [Q&A]
For some reason, all the books I added are getting listed as a series with the Authors name. Every book is doing this. Is there a way to stop it from happening? I've been going through and removing the series information one book at a time, but I have about 1,000 audiobooks collected over the years and that's a lot of manual editing. LOL
![Capture](https://user-images.githubusercontent.com/26206442/165092869-535a9058-f670-44b3-b42e-cb3dc879cdaa.jpg)

**Comment by advplyr** [comment]
What does your folder structure look like? What does your server settings page look like?

**Comment by LordSith77** [comment]
Settings Page:
![Screenshot 2022-04-25 08 02 26](https://user-images.githubusercontent.com/26206442/165094884-1f60fe0c-fbd1-4095-a9e3-f6f3d662b338.png)

Folder Structure:
![Screenshot 2022-04-25 08 02 46](https://user-images.githubusercontent.com/26206442/165094891-6703566f-7746-432b-be78-6ecef3178df3.png)

**Reply by advplyr** [reply]
I need to see the parent and child folder structure as well. Are you using OPF files? If not you can disable that.
Make sure you are following the folder structure in the docs.

**Comment by LordSith77** [comment]
![Screenshot 2022-04-25 08 12 04](https://user-images.githubusercontent.com/26206442/165096119-28b42c0a-c1c0-4363-81c2-984faf3f7232.png)
![Screenshot 2022-04-25 08 14 11](https://user-images.githubusercontent.com/26206442/165096452-dae9e5e7-f35b-43b6-815a-f4ae2c3dac14.png)

**Comment by troycarpenter** [comment]
I may be reading it wrong, but I think the problem is your directory structure.  If your "alphabetical" level is in your root directory, then your authors are showing up at the "series" level.  I'm using [https://www.audiobookshelf.org/docs/#book-structure](https://www.audiobookshelf.org/docs/#book-structure) as a reference.
